homeassistant/components/light/lifx/__init__.py

```diff
@@ -41,7 +45,10 @@
 
 CONF_SERVER = 'server'
 
+SERVICE_LIFX_SET_COLOR = 'lifx_set_color'
```

Comment:
I didn't think `lifx_set_state` carried much meaning, seeing that every service call is about changing the state. I changed it anyway, at least it does not give the wrong impression like `lifx_set_color` did.